registry/Scripts/script.sh

```diff
@@ -241,12 +242,16 @@ mkdir -p $HTPASSWD_DIR
 fetchCredentials
 cp .htpasswd $HTPASSWD_DIR/.htpasswd
 
+echo fetching available registry image tag
+REGISTRY_IMAGE_TAG=$(docker images --filter=reference='registry*:*' --format "{{.Tag}}" | head -n 1)
```

Comment:
`registry:*` is valid tighter constraint

Reply:
Let me take this up in next checkin when I will remove the tag completely.